docs/agents/scpi_psu.rst

```diff
@@ -0,0 +1,95 @@
+.. highlight:: rst
+
+.. _scpi_psu:
+
+==================
+SCPI PSU Agent
+==================
+
+This agent uses Standard Commands for Programmable Instruments (SCPI)
+It works for many power supplies, including the Keithley 2230G 
+and BK Precision 9130. It connects to the PSU over ethernet, and allows
+users to set current, voltage, and turn channels on/off. It also allows for 
+live monitoring of the PSU output. 
+
+.. argparse::
+    :filename: ../agents/scpi_psu/scpi_psu_agent.py
+    :func: make_parser
```

Comment:
This is causing a build error in the documentation. This particular line is searching the file specified in `:filename:` for a function called `make_parser`, which doesn't exist. The `make_parser` function should return the parser. You can see an example for how to move your existing parser to a `make_parser` function in several other Agents, for example the cryomech Agent.